components/brave_wallet/browser/json_rpc_service_unittest.cc

```diff
@@ -421,10 +421,10 @@ class GetAccountInfoHandler : public SolRpcCallHandler {
     std::vector<uint8_t> result(96 + data.size());
     auto result_span = base::span(result);
     // Header.
-    base::ranges::copy(owner.bytes(), result_span.subspan(32, 32).begin());
+    result_span.subspan<32, 32>().copy_from(owner.bytes());
 
     // Data.
-    base::ranges::copy(data, result_span.subspan(96).begin());
+    result_span.subspan(96u).copy_from(data);
```

Comment:
nit: `result_span.last(data.size()).copy_from(data);` should've been there in the first place

Reply:
Ah, that's better. Fixed.